Changes to build.py

jmucchiello opened the pull request:
The important change is the in the build.py file. Including an empty localization.tsv can't hurt. In .gitignore, .vs/ is very important. The other changes, less so.

jmucchiello committed:
Needed changes

jmucchiello committed:
Prevent copying examplemod.dll to the game.

Cyber28 commented on .gitignore:
```diff
@@ -1,2 +1,5 @@
\ No newline at end of file
+obj/
+.vs/
+Icon Sources/
```
This feels like a personal thing, not something that should be in the template

jmucchiello replied:
The .vs/ thing kills me every time. I had to fork the mod-template. :)

Cyber28 commented on build.py:
```diff
@@ -51,6 +51,11 @@ def sync_folder(src: Path, dst: Path):
+# check dll has unique name
+if DLL_NAME.lower() == "examplemod.dll":
+    print("Did you forget to rename the DLL in the project settings?")
+    exit(1);
```
I think a warning is enough here, maybe link to [https://modding.stacklands.co/en/latest/guides/tutorial.html#step-1-setting-up-the-template](https://modding.stacklands.co/en/latest/guides/tutorial.html#step-1-setting-up-the-template) as well

jmucchiello replied:
This is why pull requests aren't requests. Not demands. At least put something there.

Cyber28 commented on localization.tsv:
```diff
@@ -0,0 +1 @@
+Term	Notes	English
```
Not sure if including a localization file is a good choice
- People going through the tutorial could be confused by it, as the tutorial tells them to download a completely different file
- I assume most people would use a tool like Google Sheets to edit their localization files instead of a code/text editor
- Even if the above assumption is incorrect, it's easy enough to create manually thanks to the localization guide

jmucchiello replied:
I'm old school. I edit the file in visual studio directly.

jmucchiello committed:
more boilerplate

jmucchiello committed:
Update sample_food.jsont

jmucchiello committed:
LogError

jmucchiello committed:
first commit

cleanup

frequency fix

Update README.md

Revert "Update README.md"

This reverts commit 35a1fda2c294b06942c3930994451cb855c0ebba.

Revert "frequency fix"

This reverts commit 0290ac2d1b3217bceba72e379ef52abc4be8aba8.

Revert "cleanup"

This reverts commit 50815c1978eae9d9b9a1b3dc2ebd764e890d62e1.

Revert "first commit"

This reverts commit 0e9481c5e6f16b3e9fdd9e71e02798ba8f5f7d2c.

jmucchiello committed:
Update build.py

jmucchiello committed:
added Common folder

jmucchiello committed:
Update build.py

jmucchiello committed:
Update build.py

jmucchiello committed:
bbcode

jmucchiello committed:
Update Mod.csproj
